e2e-tests/src/cases.rs

```diff
@@ -55,5 +55,6 @@ pub fn possible_test_cases() -> PossibleTestCases {
             "authorities_are_staking",
             test_authorities_are_staking as TestCase,
         ),
+        ("kick_out_automatic", test_kick_out_automatic as TestCase),
```

Comment:
No corresponding change to GH workflows? Does it go to nightly?

Reply:
Added a GH action. So far, in local tests this is borderline, pushing on 10 min. If the situation does not improve on GH, I will most likely move it to nightly, but let's give it a spin first.